qemu/tests/virtio_driver_sign_check.py

```diff
@@ -48,12 +48,22 @@ def run(test, params, env):
     timeout = float(params.get("login_timeout", 240))
     session = vm.wait_for_login(timeout=timeout)
     signtool_cmd = params["signtool_cmd"]
+    drv_name = params["tested_driver"].lower()
     list_files_cmd = 'dir /s /b %s | find /i "%s" | find "%s"'
+    if "prewhql" in params["cdrom_virtio"]:
+        v_mark = "/pa"
+    else:
+        v_mark = "/kp"
 
     try:
         error_context.context("Running SignTool check test in guest...",
                               test.log.info)
         file_type = [".cat", ".sys", ".inf", "Wdf"]
+        # Add a workaround for pvpanic, as there are pvpanic-pci files
+        # include in the latest prewhql version,
+        # they are for arm support and we no need to test them currently.
+        if "pvpanic" in drv_name:
+            file_type = ["%s.cat" % drv_name, ".sys", "%s.inf" % drv_name, "Wdf"]
```

Comment:
Recommend a separated commit for this issue.

Reply:
updated, thanks~